### ISSUE TITLE
Please add to popular macOS package manager Homebrew

### ISSUE THREAD

**Issue by rnkn** [pimutils/todoman#54]
Could you please add `todoman` to [Homebrew](http://brew.sh)?

Homebrew has kinda become the de facto package manager for macOS. I'd add it myself but I don't know how....

**Comment by hobarrera** [comment]
`brew create https://github.com/pimutils/todoman/archive/v1.6.3.tar.gz` seems like a useful starting point if you're willing to look into this.

I wanted to give this a try, but most of the dependencies are missing too, so formulas for those are required as well (I only have an OS X VM laying around, I don't actively use it so as to commit to maintaining that many hombrew formulas).

**Comment by rnkn** [comment]
I'm not really familiar enough with how todoman works to create the formula for it, and it's also a bit of a chicken/egg case as I'd only become familiar with it if I could install it...

**Comment by untitaker** [comment]
You can do that using `pip install --user todoman`, which installs into `~/.local/bin` and `~/.local/lib`, though it won't update via homebrew.

**Comment by hobarrera** [comment]
> I'm not really familiar enough with how todoman works to create the formula for it

You don't really need to know any specifics, just package the dependencies are regular python libs, and then todoman.

I'll leave this open though, since contributor are welcome to package it, but since none of us use OSX/macOS, we simply can't/won't be doing this ourselves.

**Comment by rnkn** [comment]
I made an attempt at creating a formula here https://gist.github.com/rnkn/a5189ef2640595117ce8505cadd052da

But this gives me
```
Last 15 lines from /Users/Paul/Library/Logs/Homebrew/todoman/09.pip:
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/commands/install.py", line 335, in run
    wb.build(autobuilding=True)
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/req/req_set.py", line 634, in _prepare_file
    abstract_dist.prep_for_dist()
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/req/req_set.py", line 129, in prep_for_dist
    self.req_to_install.run_egg_info()
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/req/req_install.py", line 439, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/usr/local/Cellar/todoman/1.7.0/libexec/lib/python3.5/site-packages/pip/utils/__init__.py", line 707, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip.exceptions.InstallationError: Command “python setup.py egg_info" failed with error code 1 in /private/tmp/pip-9ky1pxc1-build/
```

💁‍♂️

**Comment by benjaminfrank** [comment]
Didn't see this before and I had todoman installed via homebrew so I thought this already exists. but it seems it doesn't and I must have done so in some local Formula which I deleted since then.
But I took what @rnkn did, made it work and opened a PR Homebrew/homebrew-core#9080

**Comment by hobarrera** [comment]
@benjaminfrank Thanks a bunch! Out of curiosity, it creating one virtualenv per app common on macOS?

**Comment by ilovezfs** [comment]
>common on macOS

It's Homebrew thing, not a macOS thing. See https://github.com/Homebrew/brew/pull/344.

**Comment by hobarrera** [comment]
Yes, sorry, that's what I meant, thanks.

**Comment by hobarrera** [comment]
Closed via upstream Homebrew/homebrew-core#9080.
